src/app/pages/apptrust/apptrust.page.ts

```diff
@@ -13,20 +10,55 @@ declare var navigator: any;
     styleUrls: ['./apptrust.page.scss'],
 })
 
-export class AppTrustPage implements OnInit {
-  private static readonly METRICS_KEY = 'security';
+export class AppTrustPage {
+  public serverUrl: string[];
 
-  ngOnInit() {
+  constructor(private securityService: SecurityService, private alertController: AlertController) {
+    this.serverUrl = this.securityService.getConfig();
   }
 
-  public isAvailable() {
-      // return this.platform.is('cordova');
-      return true;
-  }
-
-
-  public ionViewWillEnter(): void {
-    if (this.isAvailable()) {
-    }
+  public clientInit() {
+    this.securityService.clientInit()
+    .then(clientData => {
+      if (clientData.data.disabled) {
+        this.alertController.create({
```

Comment:
there is a bit of duplicated code here for creating the alert. This could be split into a separate function which just returns the promise you could use in the .then.I am happy to do some pair programming here if you want.

Reply:
Taught it was a bit ugly myself , Wondered who would point it out .  :) I'm ok with the refactor